helm/ingress-azure/templates/clusterrole.yaml

```diff
@@ -65,4 +65,22 @@ rules:
   verbs:
     - create
     - patch
+- apiGroups:
+    - "acn.azure.com"
+  resources:
+    - "overlayextensionconfigs"
```

Comment:
does the crd for this need to be installed on the cluster prior to use of this feature or install of AGIC (with this feature) ?